test/OpenSpaceToolkit/Astrodynamics/Trajectory/State.test.cpp

```diff
@@ -77,42 +93,166 @@ TEST(OpenSpaceToolkit_Astrodynamics_Trajectory_State, EqualToOperator)
 {
     {
         const Instant instant = Instant::DateTime(DateTime(2018, 1, 1, 0, 0, 0), Scale::UTC);
-        const Position position = Position::Meters({1.2, 3.4, 5.6}, Frame::GCRF());
-        const Velocity velocity = Velocity::MetersPerSecond({7.8, 9.0, 1.2}, Frame::GCRF());
+        VectorXd coordinates(6);
+        coordinates << 1.0, 2.0, 3.0, 4.0, 5.0, 6.0;
+        const Shared<const CoordinatesBroker> brokerSPtr = CoordinatesBroker::FromSubsets(
+            {CartesianPosition::ThreeDimensional(), CartesianVelocity::ThreeDimensional()}
+        );
 
-        const State state = {instant, position, velocity};
+        const State state_1 = {instant, coordinates, Frame::GCRF(), brokerSPtr};
 
-        EXPECT_TRUE(state == state);
+        const State state_2 = {instant, coordinates, Frame::GCRF(), brokerSPtr};
+
+        EXPECT_TRUE(state_1 == state_2);
```

Comment:
```suggestion
        EXPECT_TRUE(aState == anotherState);
```

Reply:
When possible use camelCase naming over snake_case in OSTk. Also avoid doing _1, _2. If required I believe we do:
firstState,
secondState,
thirdState etc.